src/components/test-finalisation/d255/d255.scss

```diff
@@ -0,0 +1,22 @@
+d255 {
```

Comment:
shouldnt this be :host

Reply:
sorry tom, think your confusing with des4